client/coral-admin/src/routes/Community/containers/Community.js

```diff
@@ -90,5 +66,22 @@ export default compose(
   connect(mapStateToProps, mapDispatchToProps),
   withSetUserStatus,
   withRejectUsername,
-  withData,
+  withFragments({
```

Comment:
This should not be defined as fragments but as a query, because we only want to load the whole community data only when we are on the community page.